examples/storage-examples/ceph-examples/MYSQL.md

```diff
@@ -0,0 +1,124 @@
+Setting Up and Validating Containerized MySQL
+=============================================
+
+First, OSE's Security Context Constraints (SSC) need to be defined such that the *seLinuxContext* and *runAsUser* values are set to "RunAsAny". Selinux is still enabled/enforcing on the OSE-master and OSE-nodes, as decribed in the [OSE environment readme](ENV.md).
+
+The mysql example uses the official mysql image found [here](https://hub.docker.com/_/mysql/). First, pull down the image on *each* OSE-nodes:
+
+```
+#on *each* OSE-node:
+$ docker pull mysql
+```
+
+Next, test to ensure that mysql can be run in a container:
+
+```
+$ docker run --name mysql -e MYSQL_ROOT_PASSWORD=foopass -d mysql
+```
+
+Note: to re-run the above first remove the archived container so that the container name, "mysql", can be reused:
+
+```
+$ docker ps -a
+$ docker stop <mysql-container-ID>
```

Comment:
you can use `"mysql"` as `<mysql-container-ID>`, as you set the container name with `docker run --name mysql`